core/scripts.py

```diff
@@ -2804,46 +2798,37 @@ def do_run(self, cmd_args: list[str] | None = None) -> None:
 
     def process_task(self, task: DeferredTask):
         data = InventoryReportTaskData(**task.data)
-        files = []
+        files: list[IO] = []
```

Comment:
Minor: Since we only have a single file now, does this really need to be a list?

Reply:
We still have multiple files: holds and inventory.